### DISCUSSION TITLE
Breakpoints inside <Style> breaking sveltekit app with "Semicolon or block is expected"

### DISCUSSION THREAD

**Discussion by Mrbeyond** [tailwindlabs/tailwindcss#10331] [Help]
**What version of Tailwind CSS are you using?**

"tailwindcss": "^3.2.4",

**What build tool (or framework if it abstracts the build tool) are you using?**

: postcss 8.4.21, @sveltejs/kit: 1.0.0, vite 4.0.0,

**What version of Node.js are you using?**

 v18.9.0

**What browser are you using?**

For example: Chrome, Opera

**What operating system are you using?**

For example: Linux Ubuntu

**Reproduction URL**

https://stackblitz.com/edit/sveltejs-kit-template-default-fsnfkn?file=src/routes/+layout.svelte

**Describe your issue**

Whenever a breakpoint responsiveness like sm:text-red-500 lg:text-blue-500 is used with @apply, [ Semicolon or block is expected ] error is thrown.

**Comment by adamwathan** [comment]
Hey! Make sure you set `lang="postcss"` on any style blocks that you need processed with PostCSS:

```html
<style lang="postcss">
  .tailwind-test {
    @apply bg-red-500 sm:bg-blue-500;
  }
</style>
```

That will fix the error for you 👍

**Reply by adamwathan** [reply]
Updated our docs to try and prevent anyone else from hitting this snag, thanks for the heads up!

<img width="1124" alt="image" src="https://user-images.githubusercontent.com/4323180/212550947-fa6d2f58-02d9-4a87-a094-2464fb258a70.png">
